<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookmarksViewForm</class>
 <widget class="QWidget" name="BookmarksViewForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolBox" name="toolbox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>514</width>
          <height>312</height>
         </rect>
        </property>
        <attribute name="label">
         <string>IPFS marks</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="bookmarkLine"/>
             </item>
             <item>
              <widget class="QComboBox" name="bookmarksBox"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTreeView" name="bookmarksTree"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>514</width>
          <height>312</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Follow</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="galacteek.qrc"/>
 </resources>
 <connections/>
</ui>
